### DISCUSSION TITLE
Would we appreciate a Codeowners file?

### DISCUSSION THREAD

**Discussion by wULLSnpAXbWZGYDYyhWTKKspEQoaYxXyhoisqHf** [berops/claudie#526] [Polls]
On the topic of Codowners file, an [issue](https://github.com/Berops/claudie/issues/511) has been open recently.

Do you think we would appreciate the existence of a Codeowners file in this repo?

**Comment by bernardhalas** [comment]
The response is quite clear, CODEOWNERS not required at this stage. We might revisit this in a mid-term again.
